handler/handler.go

```diff
@@ -220,16 +218,16 @@ func AddValidVoters(res http.ResponseWriter, req *http.Request) {
 // Requires admin privileges
 // Endpoint: PUT /api/voters/remove
 func RemoveValidVoters(res http.ResponseWriter, req *http.Request) {
-	if !isAuthenticatedWithPermissions(res, req, []string{"modifyAdmin"}) {
+	perms := []string{util.MODIFY_ADMIN}
```

Comment:
Took me a second to understand that perms stand for permissions, I don't think it is the most intuitive abbreviation.